[2023-03-29 15:55:39,597] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.end_task scheduled__2023-03-28T10:05:00+00:00 [queued]>
[2023-03-29 15:55:39,610] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.end_task scheduled__2023-03-28T10:05:00+00:00 [queued]>
[2023-03-29 15:55:39,610] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:55:39,610] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:55:39,610] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:55:39,630] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): end_task> on 2023-03-28 10:05:00+00:00
[2023-03-29 15:55:39,635] {standard_task_runner.py:52} INFO - Started process 168 to run task
[2023-03-29 15:55:39,637] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'end_task', 'scheduled__2023-03-28T10:05:00+00:00', '--job-id', '606', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmp2ulxoy26', '--error-file', '/tmp/tmpppf2282w']
[2023-03-29 15:55:39,637] {standard_task_runner.py:80} INFO - Job 606: Subtask end_task
[2023-03-29 15:55:39,686] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.end_task scheduled__2023-03-28T10:05:00+00:00 [running]> on host 4513f77b4fd8
[2023-03-29 15:55:39,762] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=end_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-28T10:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-28T10:05:00+00:00
[2023-03-29 15:55:39,763] {logging_mixin.py:115} INFO - DAG is completed...
[2023-03-29 15:55:39,763] {python.py:173} INFO - Done. Returned value was: DAG is completed...
[2023-03-29 15:55:39,792] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=end_task, execution_date=20230328T100500, start_date=20230329T155539, end_date=20230329T155539
[2023-03-29 15:55:39,810] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:55:39,846] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
